src/Components/test/E2ETest/ServerExecutionTests/ComponentHubInvalidEventTest.cs

```diff
@@ -16,7 +16,7 @@
 
 namespace Microsoft.AspNetCore.Components.E2ETest.ServerExecutionTests;
 
-[QuarantinedTest("https://github.com/dotnet/aspnetcore/issues/19666")]
+[Trait("QuarantinedTest", "https://github.com/dotnet/aspnetcore/issues/19666")]
```

Comment:
May be possible to avoid via https://github.com/xunit/samples.xunit/tree/main/TraitExtensibility. 👀 into it.